src/components/dialogs/limits/limits-properties-stack.tsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2025, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import { Avatar, Stack } from '@mui/material';
+import { LimitsProperty } from '../../../services/network-modification-types';
+import { useMemo } from 'react';
+import { LimitsTagChip } from './limits-tag-chip';
+import { useWatch } from 'react-hook-form';
+
+const MAX_PROPERTIES_TO_RENDER: number = 2;
+
+export interface LimitsPropertiesStackProps {
+    formName: string;
+}
+export function LimitsPropertiesStack({ formName, ...props }: Readonly<LimitsPropertiesStackProps>) {
+    const limitsProperties: LimitsProperty[] | undefined = useWatch({ name: formName });
+    const propertiesToRender: LimitsProperty[] = useMemo(() => {
+        if (!limitsProperties) {
+            return [];
+        }
+        return limitsProperties.length < MAX_PROPERTIES_TO_RENDER ? limitsProperties : limitsProperties?.slice(0, 2);
+    }, [limitsProperties]);
+
+    return (
+        <Stack direction="row" {...props}>
+            {propertiesToRender.map((property: LimitsProperty) => (
+                <LimitsTagChip key={`${property.name}$${property.value}`} limitsProperty={property} />
```

Comment:
I think it changed and we do not need to use both name and value right ? the name is unique already by limit set